tracing/init.go

```diff
@@ -42,8 +42,8 @@ func InitTracer(serviceName, instanceName string) (trace.Tracer, func(), error)
 // TracerProvider will also use a Resource configured with all the information
 // about the application.
 func newTracerProvider(serviceName, instanceName string) (*tracesdk.TracerProvider, error) {
-	// Create the Jaeger exporter
-	exp, err := jaeger.New(jaeger.WithCollectorEndpoint(jaeger.WithEndpoint(JaegerCollectorURL)))
+	// Create the OTEL exporter
+	exp, err := otel.New(context.Background(), otel.WithEndpointURL(JaegerCollectorURL))
```

Comment:
I think we need to get rid of `JaegerCollectorURL`. Right now the help still prints this:
```
  -j string
    	address of Jaeger collector to submit spans (default "http://localhost:14268/api/traces")
```
Since we only want to support OTLP, there is no need for this setting. The OTLP endpoint can be configured via OTEL env variable (need to test that it works and add to README)

Reply:
@yurishkuro I have used the env variable here. To use it currently I am exporting that env variable in my terminal before `go run main.go.`